modules/client-configuration/pages/contact-methods-saltbundle.adoc

```diff
@@ -11,7 +11,7 @@ It is also possible to use the Salt Bundle on systems that run a Salt Minion con
 
 == Client Registration with Salt Bundle as a Minion
```

Comment:
@keichwa it's actually not a method.
It's better not to remove the Minion from this part. It could make some mess in the terms, but the Salt Clients could be different ones for example the minion connected with ZeroMQ or Salt SSH client which is not actually  the minion according to the terms of Salt, but this header is to the part describing the Minion only and this part makes no sense for Salt SSH or Salt client in general.
I would suggest the following:
```suggestion
== Client Registration with the Salt Bundle as a Minion
```

Reply:
I see your point.  In the past, they told us that we'd better avoid that term. I personally cannot judge about it --I'm neither a native speaker nor an expert in SF or where ever those strangers may appear `?-)`  Thus it might make perfect sense the way you say.